modules/ting_search_carousel/ting_search_carousel.admin.inc

```diff
@@ -276,24 +276,26 @@ function ting_search_carousel_query_form($item = array(), $index = 0) {
     '#default_value' => isset($item['query']) ? $item['query'] : '',
   );
 
-  $form['data']['remove'] = array(
-    '#type' => 'submit',
-    '#value' => t('Remove'),
-    '#name' => 'ting_search_carousel_remove_' . $index,
-    '#search_id' => $index,
-    '#attributes' => array(
-      'class' => array(
-        'remove',
+  if ($index != 0) {
```

Comment:
From   if (empty($form_state['num_searches'])) { to   if (!isset($form_state['num_searches'])) {